SMLHelper/Assets/ModPrefab.cs

```diff
@@ -71,12 +72,28 @@ protected ModPrefab(string classId, string prefabFileName, TechType techType = T
         internal GameObject GetGameObjectInternal()
         {
             GameObject go = GetGameObject();
-
-            if (go == null)
-            {
+            if (!go)
```

Comment:
Please don't use this syntax for null checks.  
Keep it as `go == null` or `go != null`.

There's no need to add this potentially confusing syntax for what is likely no real benefit to the compiler.